package.json

```diff
@@ -144,5 +144,6 @@
     "pretty-format": "^29",
     "tsx": "^3.12.7",
     "typescript": "5.1.6"
-  }
+  },
+  "packageManager": "yarn@1.22.19"
```

Comment:
@JamesHenry I'll defer to you as #6162 is upcoming

Reply:
that shouldn't really affect this pr